APITestingFramework/Features/Project/GetAll.feature

```diff
@@ -0,0 +1,13 @@
+Feature: Retrieve all projects
+
+    Scenario: Retrieve all projects with valid authentication
+        Given the user is authenticated
+        When the user sends a GET request to the "/projects" endpoint
+        Then the response should have a status code of 200
+        And the response should contain a list of all projects
```

Comment:
And the response should contains "2" elements
And the response have the following projects:
  | Project Name|
   |project 1       |
   |project 2       |